CHANGELOG.md

```diff
@@ -31,6 +31,7 @@
   * `cortex_alertmanager_nflog_maintenance_errors_total`
   * `cortex_alertmanager_silences_maintenance_total`
   * `cortex_alertmanager_silences_maintenance_errors_total`
+* [ENHANCEMENT] Alertmanager: Add configuration option to enable/disable deletion of alertmanager state from object storage. This is useful when migrating alertmanager tenants from one cluster to another, as it avoids a condition where the state object is copied but then deleted before the configuration object is copied. #4989
```

Comment:
```suggestion
* [ENHANCEMENT] Alertmanager: Add configuration option to enable or disable the deletion of alertmanager state from object storage. This is useful when migrating alertmanager tenants from one cluster to another, because it avoids a condition where the state object is copied but then deleted before the configuration object is copied. #4989
```